src/server/BaseHttpHandler.java

```diff
@@ -0,0 +1,33 @@
+package server;
+
+import com.sun.net.httpserver.HttpExchange;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+public class BaseHttpHandler {
+
+    protected void sendText(HttpExchange h, String text, int code) throws IOException {
```

Comment:
Давай переименуем однобуквенную переменную `h`:

```suggestion
    protected void sendText(HttpExchange httpExchange, String text, int code) throws IOException {
```